hubspot.com.hubspot-template-3.json

```diff
@@ -8,12 +8,14 @@
     "logoUrl": "https://domainconnect.org/wp-content/uploads/2017/02/HubSpot-768x223.png",
     "description": "Enables an apex domain to work with the HubSpot CMS",
     "variableDescription": "IPs: HubSpot CMS IPs",
-    "records": [{  
+    "records": [{
+         "groupId":"a1",
          "type":"A",
          "host":"@",
          "pointsTo":"%IP1%",
```

Comment:
let's be consistent with naming? how about "%ip1Value%?

Reply:
That is an existing template;
Renaming variable can potentially break something